.idea/vcs.xml

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project version="4">
+  <component name="VcsDirectoryMappings">
+    <mapping directory="$PROJECT_DIR$" vcs="Git" />
+  </component>
+</project>
```

Comment:
Do we necessary need to add these IntelliJ dot files to the project or are these .idea/* files are by mistake committed?

Reply:
thanks for spotting this, this is from my IDE (Goland) and should go to `.gitignore`